amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -434,3 +434,23 @@ type ColonyAction @model {
   toDomain: Domain @hasOne(fields: ["toDomainId"])
   fundamentalChainId: Int
 }
+
+type ParsedMotionStakedEventValues {
+  amount: String!
+  motionId: String!
+  stakeAmount: String!
+  staker: String!
+  vote: Int!
+}
+
+type ParsedMotionStakedEvent {
```

Comment:
So the parsed naming scheme was used for data that was post-processed by Apollo's resolvers (back in the Dapp).

Since that is no longer the case, I think we should drop the `Parsed` part of the naming scheme